packages/cli/src/handlers/preview.ts

```diff
@@ -338,7 +338,7 @@ export const stopPreviewHandler = async (
         });
 
         console.error(
-            `Could not find preview project with name ${projectName}`,
+            styles.error(`You don't have access to this resource or action`),
```

Comment:
The ticket was referring to how the error message was displayed ( eg: red colour ) and not about the message itself.

Reply:
Thanks @ZeRego, let me go through it again.